olm_deploy/scripts/operator-uninstall.sh

```diff
@@ -9,6 +9,7 @@ oc delete --wait --ignore-not-found ns ${CLUSTER_LOGGING_OPERATOR_NAMESPACE}
 oc delete --wait --ignore-not-found crd clusterloggings.logging.openshift.io
 oc delete --wait --ignore-not-found crd collectors.logging.openshift.io
 oc delete --wait --ignore-not-found crd logforwardings.logging.openshift.io
+oc delete --wait --ignore-not-found crd clusterlogforwarders.logging.openshift.io
```

Comment:
do we still need the delete of logforwardings or is that dead code now?

Reply:
For a N+1 release we will need it, since the CRD will be in place for people to migrate from tech preview to GA.